pkgs/top-level/python-packages.nix

```diff
@@ -11176,6 +11176,29 @@ in {
 
   keyring = callPackage ../development/python-modules/keyring { };
 
+  kivy = callPackage ../development/python-modules/kivy { inherit stdenv buildPythonPackage maintainers platforms licenses pkgs self ; };
+
+  kivygarden = buildPythonPackage rec {
+    # The sole reason for this package is that it is a (questionable) dependency of Kivy
+    name = "kivy-garden-0.1.4";
+
+    src = pkgs.fetchurl {
+      url = "mirror://pypi/k/kivy-garden/${name}.tar.gz";
+      sha256 = "0wkcpr2zc1q5jb0bi7v2dgc0vs5h1y7j42mviyh764j2i0kz8mn2";
+    };
+
+    buildInputs = with self; [ requests ];
```

Comment:
Is this not needed during runtime? Or is it needed for the tests? In case of the latter, use `checkInputs`.

Reply:
It's needed during runtime.

kivy-garden is a "not invented here" package manager for kivy components, and as such uses `requests` to fetch those components. 

The real problem is that such a thing exists in the first place, and that it is an explicit dependency of Kivy. I'll talk with the kivy people about both of those issues.

In the meantime, given the fact that the present PR makes kivy-garden available, it should do so in a non-broken way, so I'll promote this to a `propagatedBuildInput`